### ISSUE TITLE
Smash Icon vs. blank image

### ISSUE THREAD

**Issue by tblock007** [tblock007/PRImageGenerator#5]
When Ron generates the character map from the spreadsheet database, some players are in the character map, but do not have a character for the specified game.  As a result, some players, if the character played is unknown, have a Smash icon next to their name, and some have no icon.

Action items are to determine the exact conditions that lead to smash icon or no icon, and update the program so that it always gives no icon.

In other words, the Smash icon should ONLY appear in the image if the character map contains an unrecognized (non-empty) string (that is also not the name of a custom image).
